integration-tests/field-validation/tests/update-example.exotest

```diff
@@ -0,0 +1,33 @@
+stages:
+  - operation: |
+        mutation {
+          updateExample(id: 1, data: { limit: 0 }) {
```

Comment:
Adding `age` here with values outside the `limit` range will make these tests even better